locales/en/client.json

```diff
@@ -798,6 +798,7 @@
   "ERROR.ErrorCloningWorld": "Error cloning world: {{error}}",
   "ERROR.ErrorRetrievingData": "Error retrieving data: {{error}}",
   "ERROR.FailedToCreateGame": "Failed to create game: {{error}}",
+  "ERROR.FailedToCreateGame.NoSystemLoaded": "Failed to create game. Please select a system or world.",
```

Comment:
I stand corrected! There's a typo here - the key you used in the component is `ERROR.FailedToCreateGame.NoSystemSelected`.

Reply:
This'll teach me to submit a pull request before a nap :facepalm: Should be taken care of now.